java-manta-client-kryo-serialization/pom.xml

```diff
@@ -101,6 +102,10 @@
                     <groupId>com.esotericsoftware</groupId>
                     <artifactId>minlog</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Why have you excluded the objenesis library? I was under the impression that kryo needed it to run.